practice-one/src/components/CustomerAvatar/CustomerAvatar.css

```diff
@@ -0,0 +1,23 @@
+.customer__wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.customer__avatar {
+  width: 30px;
+  height: 30px;
+  border-radius: 100%;
```

Comment:
border-radius 50% is enough

Reply:
I changed as you commented
![image](https://github.com/datdragon777/React-Training/assets/74806826/f413d10e-b345-45d9-a108-f4edc391a170)